kube-derive/src/custom_resource.rs

```diff
@@ -1,6 +1,6 @@
 // Generated by darling macros, out of our control
 #![allow(clippy::manual_unwrap_or_default)]
-use darling::{FromDeriveInput, FromMeta};
+use darling::{util::Override, FromDeriveInput, FromMeta};
```

Comment:
huh, interesting [new thing](https://docs.rs/darling/latest/darling/util/enum.Override.html). maybe that has other uses. the base example there looks basically like our crate path stuff 🤔

Reply:
Yes, it is really neat. I actually also had the same thing on my mind which you just mentioned. I hinted looking to introduce a bunch of other changes across the derive macro here already: https://github.com/kube-rs/kube/pull/1656#discussion_r1929538068

So I'm happy to explore this and other ideas further in a future PR.